styles/hydrogen.less

```diff
@@ -79,7 +83,8 @@
 
         .bubble-output-container {
             padding: 3px;
-            overflow: scroll;
+            padding-right: 10px;
+            overflow: auto;
         }
```

Comment:
Yes scrolling works.
For me the only difference is that the space for the scrollbar isn't reserved with `auto` when no scrolling is needed. I'm talking about these dark bars:
<img width="390" alt="bildschirmfoto 2016-02-26 um 11 58 28" src="https://cloud.githubusercontent.com/assets/13285808/13350441/5c131032-dc80-11e5-91be-c38aca6c44b8.png">